rspirv/sr/autogen_ops.rs

```diff
@@ -768,9 +768,6 @@ pub enum Op {
         semantics: spirv::Word,
         value: spirv::Word,
     },
-    Phi {
```

Comment:
I'm beginning to feel that our approach to autogen is flawed, considering how many changes need to be made beyond the base information provided in the SPIR-V json.

Reply:
We had this discussion earlier in the SR path with @antiagainst , and it wasn't clear if it was worth it to just hand-write all the instructions manually. We decided to proceed with autogen modifications until it's more clear. So let's indeed keep an eye on the amount of exceptions and custom handling in the autogen.